package.json

```diff
@@ -75,6 +75,21 @@
                     "type": "string",
                     "default": "^8.47.0",
                     "description": "%extension.commands.salesforce-mobile-offline.eslint.version%"
+                },
+                "salesforce.salesforcedx-vscode-mobile.diagnostics.suppressAll": {
```

Comment:
These names should be trimmed, as they translate directly into titles and other descriptive verbiage in the settings UI. Taking some cues from `properties` section of [Configuration schema](https://code.visualstudio.com/api/references/contribution-points#Configuration-schema), it feels like a better name for this property would be:

```suggestion
                "salesforcedx-vscode-mobile.diagnostics.suppressAll": {
```

Which I _think_ should translated in the UI to:

# Salesforce Mobile Extensions

_Other settings for the extension_

**Diagnostics: Suppress All**
_Description, input, etc._

Reply:
thanks @khawkins giving the wording for it.  now it's below. 
<img width="446" alt="image" src="https://github.com/user-attachments/assets/e3e2bed9-4d90-4573-8d85-0fa52c879b37">